code/src/components/Loading.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import styled, { keyframes } from 'styled-components';
```

Comment:
Very nice loading page, impressive that you created the Icon yourself and didnt use an image!